src/Authoring/WinRT.SourceGenerator/Generator.cs

```diff
@@ -203,10 +203,12 @@ public class SourceGenerator : ISourceGenerator
     {
         public void Execute(GeneratorExecutionContext context)
         {
+            /*

```

Comment:
intentional?

Reply:
Yes, it's temporary for testing, will be removed in final commit.